sensing/build.gradle.kts

```diff
@@ -64,4 +64,13 @@ dependencies {
   testImplementation(Dependencies.junit)
   androidTestImplementation(Dependencies.AndroidxTest.extJunit)
   androidTestImplementation(Dependencies.Espresso.espressoCore)
+  testImplementation(Dependencies.AndroidxTest.core)
+  testImplementation(Dependencies.Kotlin.kotlinCoroutinesTest)
+  testImplementation(Dependencies.Kotlin.kotlinTestJunit)
+  testImplementation(Dependencies.junit)
```

Comment:
nit: Looks like this is a duplicate `testImplementation(Dependencies.junit)`

Reply:
Done.